v1/personio_test.go

```diff
@@ -515,6 +515,17 @@ func TestClient_GetTimeOffs(t *testing.T) {
 			continue
 		}
 
+		for _, timeOff := range timeOffs {
+			if len(timeOff.Comment) <= 0 {
+				t.Errorf("[%d] Time-off with ID %d has empty comment", testNumber, timeOff.Id)
```

Comment:
Why does the test fail if the comment is empty? Or rather why is there an implication that the comment is required?

Reply:
Because all our test data has a comment now and it felt a little redundant to add it to the test parameterization struct.

Reply:
I just added it for completeness to ensure that, in further version of the lib, the field stays present. (The fixed bug was the omitted field.)

Reply:
Do you have a suggestion as to how to improve the test?

Reply:
Works for me - generally having enough testcases that both are handled is usually what I go for. Fine for now here.